discovery_engine_core/bindings/src/lib.rs

```diff
@@ -40,6 +40,12 @@ use std::path::Path;
 use itertools::Itertools;
 use xayn_discovery_engine_core::Engine;
 
+#[allow(unsafe_code)]
+#[no_mangle]
+pub extern "C" fn rust_cfg_storage() -> u8 {
```

Comment:
:art: what do you think about the following naming? it's implied that this comes from rust via the ffi anyways. then it would be aligned with the naming we already use in dart and rust.
```suggestion
pub extern "C" fn cfg_feature_storage() -> u8 {
```